renderdoc/driver/vulkan/vk_info.h

```diff
@@ -479,6 +479,7 @@ struct VulkanCreationInfo
       int32_t depthstencilResolveAttachment;
       int32_t fragmentDensityAttachment;
       int32_t shadingRateAttachment;
+      VkSampleCountFlagBits multisampledRenderToSingleSampledRasterizationSamples;
```

Comment:
Done. Changed to `msrtssSamples`

Reply:
I don't think we should use `msrtss` in the codebase, as I feel it's likely only immediately intuitive to those intimately familiar with the extension to identify the shortened form of the extension name. Unless I'm mistaken it's purely the (somewhat arbitrary) extension name and not a name for the technique in general. Given the many AA and supersampling acronyms in recent years I fear the acronym alone is liable to cause confusion.

If you don't want to use the 'fake' shorthand then maybe something like `subpassTemporarySampleCount` or `subpassInternalSampleCount` or even just `tileOnlyMSAASampleCount` since realistically this is only for tilers and we shouldn't really contort to pretend otherwise? At least then even if someone doesn't understand exactly what it denotes the name is at least unique and less jargony.

I'm definitely aware that this is bordering on nitpicky, but I do think naming is important particularly when it's user-facing in the pipeline viewer as you have it.

Reply:
I went with `tileOnlyMSAA` to accommodate the fact that dynamic rendering doesn't have subpasses.

Nits are good, no worries :)